hadoop-hdds/common/src/main/resources/ozone-default.xml

```diff
@@ -1620,18 +1620,27 @@
 
   <property>
     <name>ozone.om.ratis.minimum.timeout</name>
-    <value>1s</value>
+    <value>5s</value>
```

Comment:
Should this be the same as `ozone.om.ratis.leader.step.down.wait.time`? IIUC, if leader is partitioned from its peers then a new leader can be elected within 5 seconds however the old leader won't step down for 30 seconds. So there is a large interval when the old leader can be serving reads thinking that it is still the leader.

Reply:
I misunderstood leader.step.down.time. It sets the time an old leader, after stepping down, has to wait before instantiating another election. 
The leader step down on losing majority heartbeats is based on the server rpc min and max timeout only. Removed this setting in new commit.